GUI/window.py

```diff
@@ -1079,6 +1085,12 @@ class SevenSegmentDisplay(Widget):
     rightE = ListProperty([.41, .41, .41])
     rightF = ListProperty([.41, .41, .41])
     rightG = ListProperty([.41, .41, .41])
+    border = ListProperty(theme_manager.primary_color)
```

Comment:
theme manager can be accessed from app. You might need to move this inside of the `__init__` method

`border_color = ListProperty(theme_manager.primary_color)`